docs/index.asciidoc

```diff
@@ -45,6 +45,7 @@ This plugin supports the following configuration options plus the <<plugins-{typ
 | <<plugins-{type}s-{plugin}-password>> |<<password,password>>|Yes
 | <<plugins-{type}s-{plugin}-port>> |<<number,number>>|No
 | <<plugins-{type}s-{plugin}-secure>> |<<boolean,boolean>>|No
+| <<plugins-{type}s-{plugin}-save_attachments>> |<<boolean,boolean>>|No
```

Comment:
Docs are  building cleanly and generally look good.  Will you please move the option in the table to alpha order, too?

Reply:
@robbavey  Will you please review and comment on recent code changes?  Thanks!